### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by ajaycstech**
* [ ] Your changes are not possible to do through a plugin and relevant
    to a large audience (ideally all users of OctoPrint)
  * [ ] If your changes are large or otherwise disruptive: You have
    made sure your changes don't interfere with current development by
    talking it through with the maintainers, e.g. through a
    Brainstorming ticket
  * [ ] Your PR targets OctoPrint's devel branch if it's a completely 
    new feature, or maintenance if it's a bug fix or improvement of
    existing functionality for the current stable version (no PRs 
    against master or anything else please)
  * [ ] Your PR was opened from a custom branch on your repository
    (no PRs from your version of master, maintenance or devel please),
    e.g. dev/my_new_feature or fix/my_bugfix
  * [ ] Your PR only contains relevant changes: no unrelated files,
    no dead code, ideally only one commit - rebase and squash your PR 
    if necessary!
  * [ ] Your changes follow the existing coding style
  * [ ] If your changes include style sheets: You have modified the
    .less source files, not the .css files (those are generated with
    lessc)
  * [ ] You have tested your changes (please state how!) - ideally you
    have added unit tests
  * [ ] You have run the existing unit tests against your changes and
    nothing broke
  * [ ] You have added yourself to the AUTHORS.md file :)

### CODE REVIEW

**Comment by karenetheridge** [comment]
> We dislike seeing spam pull requests just as much as you, so please give them an invalid or spam label and close them. Pull requests that contain a label with the word invalid or spam won’t be counted toward Hacktoberfest.

> As a maintainer, how should I handle spam pull requests?

https://hacktoberfest.digitalocean.com/faq

File: README.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ after all :)
 For information about how to go about submitting bug reports or pull requests, please see the project's
 [Contribution Guidelines](https://github.com/OctoPrint/OctoPrint/blob/master/CONTRIBUTING.md).
 
-## Installation
+##Basic Installation
```
Comment (cp2004) [inline README.md:54]: Formatting is wrong, this is no longer a title

**Comment by cp2004** [comment]
Does look like spam to me... There isn't a specific 'spam' or 'invalid' label and I don't have the permissions to create more. 

Seems they still count towards the challenge even if they aren't merged so the only way to stop that is with the labels. 🤔

**Comment by foosel** [comment]
Marked as spam, user reported and blocked.

**Pull request closed without merging**